### ISSUE TITLE
Generating signed APK for react-native@0.56

### ISSUE THREAD

**Issue by josephsaroufim** [facebook/react-native#20973]
At react-native@0.56, generating signed apk is not possible due it's presenting a missing module 'AccessibilityInfo'
Any recommendations?
